build-storybook.log

```diff
@@ -0,0 +1,87 @@
+
+> @plyaz/ui@0.1.8 build-storybook /Users/usmanqadir/Desktop/projects/ui
+> storybook build --output-dir\=/var/folders/jf/ms82wz0s223_p2_kk60fjhfh0000gp/T/chromatic--27307-eY6DD0d4Ddnq
+
+[1m@storybook/core v8.6.12
+[22m
+info => Cleaning outputDir: [36m../../../../../var/folders/jf/ms82wz0s223_p2_kk60fjhfh0000gp/T/chromatic--27307-eY6DD0d4Ddnq[39m
+info => Loading presets
+info Found existing addon "@storybook/addon-controls", skipping.
+info Found existing addon "@storybook/addon-docs", skipping.
+info Found existing addon "@storybook/addon-viewport", skipping.
+info Found existing addon "@storybook/addon-controls", skipping.
+info Found existing addon "@storybook/addon-docs", skipping.
+info Found existing addon "@storybook/addon-viewport", skipping.
+info => Building manager..
+info => Manager built ([38;2;181;126;229m147 ms[39m)
+info => Building preview..
```

Comment:
This should be ignored and i think `bundle*` files and so on should be from a dir like `performance`, you may need to configure some settings

Reply:
I'll move them there.